VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
So choosing between, updating Makefile for updating the version or updating the version file, I think I will go for the later.  @budhrg @hferentschik WDYT?

Not sure it make a difference, but we have been doing in some of the eco systems projects for the reasons I have mentioned in https://github.com/minishift/minishift-b2d-iso/pull/16#discussion_r95108894
e.g.: 
* https://github.com/minishift/minishift/blob/master/VERSION
* https://github.com/projectatomic/vagrant-service-manager/blob/master/lib/vagrant-service-manager/version.rb
* https://github.com/boot2docker/boot2docker/blob/master/VERSION